pyxem/tests/test_utils/test_ri_utils.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+import pytest
+import numpy as np
+from pyxem.utils.ri_utils import *
+from pyxem.signals.electron_diffraction2d import ElectronDiffraction2D
+
+
+def test_as_signal_generation():
+
+    N = ElectronDiffraction2D([[np.array(1.)]])
+    C = ElectronDiffraction2D([[np.array(0.)]])
+    elements = ['Cu']
+    fracs = [1]
+    s_size = 10
+    s_scale = 0.1
+    types = ['lobato', 'xtables', 'not_implemented']
+
+    for type in types:
```

Comment:
Don't call things `type` in python, use `table_choice` or something?

Reply:
Changed.